Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -378,7 +378,7 @@ public void ProcRunner_DoNotLogSensitiveData()
                 logger.AssertSingleDebugMessageExists(arg);
             }
 
-            logger.AssertSingleDebugMessageExists(SonarQube.Common.Resources.MSG_CmdLine_SensitiveCmdLineArgsAlternativeText);
+            logger.AssertSingleDebugMessageExists("<sensitive data removed>");
```

Comment:
What's this change?

Reply:
Because of `Directory.Build.targets`, the generated resource classes are now compiled into internal classes. I thought about changing that, but decided to leave it and tweak couple of tests instead.

Reply:
It's probably better to use `InternalsVisibleTo` there.